requirements-test.txt

```diff
@@ -17,7 +17,7 @@ peft
 requests
 ray
 sentence-transformers # required for embedding
-compressed-tensors==0.4.0 # required for compressed-tensors
+compressed-tensors==0.5.0 # required for compressed-tensors
```

Comment:
you can drop this fi in common

Reply:
@robertgshaw2-neuralmagic See previous comments. Some of the CI tests only install from requirements-tests